udata-front/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I'm not sure about the archi of this folder -> now having udata_front and udata-front directories.

Reply:
I don't know of any best practice for a case like this where we have a back-end library and a front-end library in the same repository.

The idea was to show that udata_front and udata-front were 2 projects for 2 distincts languages.

Usually, NPM libraries have a `src` folder but i find it weird for udata-front because we also have the udata_front folder.

Also, `lib/` is gitignored for some reason.

Do you have another idea we can try ?

Reply:
The directory is now called udata-front-plugins-helper to match the containing NPM package